styles/home/info.css

```
@@ -31,25 +38,26 @@
 .info__content {
   display: flex;
   flex-direction: column;
-  gap: 12px;
+  gap: 1.2rem;
 }
 
 .info__image {
-  width: 579px;
-  height: 444px;
-}
-
-.info .info__content .info__category {
-  color: var(--primary-100);
-  font-size: 18px;
+  width: 57.9rem;
+  height: 44.4rem;
```

💊 제안
이미지는 원본 이미지의 비율을 유지하는 것이 중요합니다~
widht, height 둘다 작성해주시기보다 기준이 되는 속성만 남겨두시고 다른 값은 자동으로 비율에 따라 늘어나도록
작업하시는 것을 추천드려요~

```suggestion
  width: 57.9rem; // width 만 조정되면 되니 height 삭제, height는 비율에 따라 늘어남.
```